### PR TITLE
Add new files for PG13 support

### PR DESCRIPTION
**Pull request by keithf4**
# Description  

Add built-in support for PostgreSQL 13

Fixes (issue) #  

Depends on (issue) #  

## Type of change  
Please check all options that are relevant  
- [ ] Bug fix (change which fixes an issue)  
- [x] Feature (change which adds functionality)  
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)  
- [ ] Documentation update only  

# How Has This Been Tested?  

**Tested Configuration**:  
- [x] CentOS, Specify version(s):  7
- [x] PostgreQL, Specify version(s):  13
- [x] docs tested with hugo >0.60  

Tested with playbook(s):  
# Checklist:  
- I have made corresponding changes to:  
    - [x] the documentation  
    - [x] the release notes  
    - [x] the upgrade doc

### CODE REVIEW

**Comment by keithf4** [comment]
Make sure to update for new blocked query time - https://github.com/CrunchyData/pgmonitor/pull/207

File: exporter/postgres/crunchy-postgres-exporter-pg13-el6.service [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    status)
+	check_status
+        ;;
+    restart)
+        stop
+        start
+        ;;
+    *)
+        N=/etc/init.d/${NAME}
+        echo "Usage: $N {start|stop|status|restart}" >&2
```
Comment (hunleyd) [inline exporter/postgres/crunchy-postgres-exporter-pg13-el6.service:78]: won't this work?
```suggestion
        echo "Usage: $0 {start|stop|status|restart}" >&2
```
then you can nuke line 77 entirely

**Review by hunleyd — CHANGES_REQUESTED** [review]

File: exporter/postgres/crunchy-postgres-exporter-pg13-el6.service (hunk 1 above)
Comment (hunleyd) [reply exporter/postgres/crunchy-postgres-exporter-pg13-el6.service:78]: Fair enough. Is EL6 the only service file we have?

Comment (keithf4) [reply exporter/postgres/crunchy-postgres-exporter-pg13-el6.service:78]: EL6 has a service file for each version of PG which is why I didn't want to worry about it. There is a single service file for EL7 but it's systemd so I don't think this applies there.

**Review by hunleyd — DISMISSED** [review]

File: exporter/postgres/crunchy-postgres-exporter-pg13-el6.service [hunk 2]
```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```
Comment (sharmay) [inline exporter/postgres/crunchy-postgres-exporter-pg13-el6.service:1]: Why do we have el6 service file ? We are not building PG for el6.

**Review by sharmay — CHANGES_REQUESTED** [review]

File: exporter/postgres/crunchy-postgres-exporter-pg13-el6.service (hunk 2 above)
Comment (keithf4) [reply exporter/postgres/crunchy-postgres-exporter-pg13-el6.service]: Removed PG13 el6 file

**Review by hunleyd — APPROVED** [review]

**Comment by sharmay** [comment]
**How Has This Been Tested?** and **Checklist** still pending.

**Comment by keithf4** [comment]
> **How Has This Been Tested?** and **Checklist** still pending.

Done

**Review by hunleyd — APPROVED** [review]

**Review by sharmay — APPROVED** [review]

**Pull request merged**